src/toil/batchSystems/singleMachine.py

```diff
@@ -87,6 +87,8 @@ def __init__(self, config, maxCores, maxMemory, maxDisk):
         """
         :type list[Thread]
         """
+        # An event used to tell the workers to quit
+        self.terminate = Event()
```

Comment:
https://github.com/BD2KGenomics/toil/wiki/Python-Dos-and-Don'ts#dont-use-threadingevent-to-signal-boolean-conditions-between-threads